app/views/admin/npq/applications/_application_details.html.erb

```diff
@@ -1,18 +1,33 @@
+<h2 class="govuk-heading-m">Application details</h2>
 <%=
   govuk_summary_list do |sl|
     sl.row do |row|
-      row.key(text: 'Application UUID')
+      row.key(text: 'Application ID')
       row.value(text: application.id || govuk_tag(text: "Missing", colour: "red"))
     end
 
     sl.row do |row|
-      row.key(text: 'User UUID')
+      row.key(text: 'User ID / Participant ID')
       row.value(text: application.user.id || govuk_tag(text: "Missing", colour: "red"))
     end
 
     sl.row do |row|
-      row.key(text: 'User Identity UUID')
-      row.value(text: application.user.id || '-')
+      row.key(text: 'GAI ID')
+      row.value(text: application.participant_identity.id || '-')
+    end
+
+    sl.row do |row|
```

Comment:
@peteryates This is saw in the participants data so thought makes sense for the `External ID`

Reply:
Makes sense to me.